relay-server/src/services/processor.rs

```diff
@@ -1274,12 +1274,22 @@ impl EnvelopeProcessorService {
         event::finalize(state, &self.inner.config)?;
         self.light_normalize_event(state)?;
         dynamic_sampling::normalize(state);
-        event::filter(state)?;
+        let global_config = self.inner.global_config.current();
+        event::filter(state, global_config.filters())?;
```

Comment:
I think I prefer `event::filter` takes a reference to the entire global config and then just picks the filters out of it, gives us a little bit more type safety.

Reply:
`relay-dynamic-config` depends on `relay-filter` so passing the full configs isn't an option and we're limited to just passing the versions, unfortunately.

Reply:
Isnt `event::filter` in `relay-server`?

Reply:
Oh right hehe yeah I agree, and applied in [8384897](https://github.com/getsentry/relay/pull/3161/commits/83848978c571ce5ad9a82cb47064a3bebc318a2a).